src/mol-plugin-ui/viewport.tsx

```diff
@@ -126,6 +130,7 @@ export class ViewportControls extends PluginUIComponent<ViewportControlsProps, V
                 <div>
                     <div className='msp-semi-transparent-background' />
                     {this.plugin.config.get(PluginConfig.Viewport.ShowControls) && this.icon(BuildOutlinedSvg, this.toggleControls, 'Toggle Controls Panel', this.plugin.layout.state.showControls)}
+                    {this.plugin.config.get(PluginConfig.Viewport.ShowSecondaryStructureSequence) && this.icon(BuildOutlinedSvg, this.toggleSSSequence, 'Toggle Secondary Structure Sequence', this.plugin.layout.state.showSecondaryStructureSequence)}
```

Comment:
How about moving the toggle here? I think it makes more sense to show it directly in the control.

![image](https://github.com/molstar/molstar/assets/21145776/5be8c3e5-d5d8-49b3-8d4e-a7ceaafe81b6)